File: iniciante/java/1015/1015.java
//QUESTÃO 1015 - Distância entre dois Pontos

import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
 	        Scanner sc = new Scanner(System.in);
			            
	        double x1 = sc.nextDouble();
	        double y1 = sc.nextDouble();
					            
 	        double x2 = sc.nextDouble();
	        double y2 = sc.nextDouble();
							            
		double distancia = Math.sqrt((Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2)));
								     
	    	String saida = String.format("%.4f", distancia);
	        System.out.println(saida);
 		sc.close();
	}
}

File: iniciante/c99/1015/1015.c
#include <stdio.h>
#include <math.h>

int main() {
	
	double x1, x2, y1, y2;

	scanf("%lf", &x1);
	scanf("%lf", &y1);
	scanf("%lf", &x2);
	scanf("%lf", &y2);

	double distancia = sqrt(pow((x2 - x1), 2) + pow ((y2 - y1), 2));

	printf("%.4lf\n", distancia);

	return 0;


}
